docs/tutorials/simple_logging_and_checkpointing.rst

```diff
@@ -0,0 +1,238 @@
+Simple Logging and Checkpointing recipe in Modulus
```

Comment:
Also, would be good to remove `in Modulus` in all these titles. This is the Modulus documentation anyway, so this is redundant and kinda distracting.

Reply:
Removed